wallet/src/types.rs

```diff
@@ -40,6 +40,8 @@ pub type SessionWallet = Arc<repository::Wallet<db::EncryptedDb>>;
 
 pub type Wallet = repository::Wallet<db::EncryptedDb>;
 
+pub type BirthDateParam = Option<u32>;
```

Comment:
This type is useless, just use `Option<Epoch>` everywhere instead.

Reply:
+1